azext_iot/digitaltwins/params.py

```diff
@@ -456,6 +456,14 @@ def load_digitaltwins_arguments(self, _):
             "In case of an error processing a batch, the behavior is determined by the --failure-policy parameter. ",
             arg_group="Models Input",
         )
+        context.argument(
+            "max_models_per_batch",
+            options_list=["--max-models-per-batch", "--mmpb"],
+            help="Indicates the maximum model size per batch when create models using directory contains "
+            "more than 250 models. The default batch size will be 30. This value could be useful when encouter "
+            "DTDLParser error. Reduce the number to prevent models pulling disordered from directory. ",
```

Comment:
The maximum model size per batch when creating more than 250 models. Reduce this number to prevent a DTDLParser error.

default should be shown automatically when you have a default argument in the command params (try running the command with -h)

also not sure what the last sentence (Reduce the number to prevent models pulling disordered from directory) means but we should put that into help if it is important

Reply:
I like the description suggestion in the first sentence of @vilit1's comment, its clear and concise.

Regarding the last sentence, the client should be sending models in the right order, dependencies of models should be in the same or prior batch. If client side processing has issues, we should fix it - however our prior understanding was a service side processing lag issue.